### PR TITLE
space fighter

### PR DESCRIPTION
**Pull request by Abdullah-Usman**
Author name

Author: Abdullah Usman
About your game

What is your game about?

The game is a space shooter where the player controls a spaceship (p) and engages in combat with enemy spaceships (e). The player can shoot lasers (l) and has to avoid collisions with enemies. The game features levels, scoring, and sound effects.

How do you play your game?

    Press A to move the player spaceship left.
    Press D to move the player spaceship right.
    Press I to shoot lasers.
    Press J to restart the game.

### CODE REVIEW

**Review by Abdullah-Usman — COMMENTED** [review]
abdullahusman

File: games/space fighter.js
Comment (Abdullah-Usman) [inline games/space fighter.js:1]: a

File: canvas.png
Comment (Abdullah-Usman) [inline canvas.png:1]: a

**Comment by Abdullah-Usman** [comment]
g

**Review by Abdullah-Usman — COMMENTED** [review]
reviewed

**Review by Abdullah-Usman — COMMENTED** [review]
team-review-requested:vercel

**Review by LucasHT22 — CHANGES_REQUESTED** [review]
Thanks for PR! But this game is VERY similar to [Alien_Attack](https://sprig.hackclub.com/gallery/Alien_Attack), you're welcome to remix games, but please make sure to state the original author and modify it moderately.

If you need help, feel free to ask:)

**Pull request closed without merging**